modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/VisibleSelectInput.es.js

```diff
@@ -89,6 +105,7 @@ const VisibleSelectInput = forwardRef(
 				onClick={onClick}
 				onKeyDown={onKeyDown}
 				ref={ref}
+				tabIndex="1"
```

Comment:
@joaovictoralvesmartins, is it really necessary? The `div` below already has the property `tabindex`, can you please clarify?

Reply:
Hello @natocesarrego !
You were right. The tabindex="1" wasn't necessary. Just pushed another version of the code.